packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -189,8 +229,9 @@ export class CppBuildConfigurationChanger implements QuickOpenModel {
     }
 
     open() {
+        const configs = this.cppBuildConfigurations.getConfigs();
         this.quickOpenService.open(this, {
-            placeholder: 'Choose a build configuration...',
+            placeholder: (configs.length) ? 'Choose a build configuration...' : 'No build configurations present',
```

Comment:
Really small nit.  If you only have one entry in settings.json, that still contains the placeholder values, it won't be displayed in the menu.  However, the message shown here will be `'Choose a build configuration...'` rather than `'No build configurations present'`.  The message `'No build configurations present'` should probably be displayed when there are no build configs displayed